components/auto-complete/style/index.less

```diff
@@ -44,7 +44,7 @@
     }
 
     .@{input-prefix-cls} {
-      background: transparent;
```

Comment:
`background: transparent;` could be useful, we can't change to white.

Reply:
same as other `transparent` changes.

Reply:
Perhaps we can change value of `@input-bg` in default theme to `transparent`? Should allow more maximum use-cases / flexibility since default `@component-background` is already `#fff`